apps/meteor/app/livechat/server/lib/LivechatTyped.ts

```diff
@@ -773,17 +776,20 @@ class LivechatClass {
 
 			if (result.status === 200) {
 				metrics.totalLivechatWebhooksSuccess.inc();
-			} else {
-				metrics.totalLivechatWebhooksFailures.inc();
+				return result;
 			}
-			return result;
+
+			metrics.totalLivechatWebhooksFailures.inc();
+			throw new Error(await result.text());
 		} catch (err) {
-			Livechat.webhookLogger.error({ msg: `Response error on ${11 - attempts} try ->`, err });
+			const retryAfter = timeout * 4;
+			Livechat.webhookLogger.error({ msg: `Error response on ${11 - attempts} try ->`, err });
```

Comment:
```suggestion
			Livechat.webhookLogger.error({ msg: `Error response on ${11 - attempts} try of omnichannel webhooks ->`, err });
```

Reply:
hmmm not sure about this one. The other one was commited already.